src/test/java/com/digitalbank/tests/LoginTests.java

```diff
@@ -54,7 +57,9 @@ public void validLoginTest() throws InterruptedException {
         driver.findElement(By.id("username")).sendKeys(VALID_USERNAME);
         driver.findElement(By.id("password")).sendKeys(VALID_PASSWORD);
         
+        Thread.sleep(5000);
```

Comment:
@bor8687 Good use of Thread class here, no Selenium command to arbitrarily wait!